Packs/ThalesCipherTrustManager/Integrations/ThalesCipherTrustManager/ThalesCipherTrustManager.py

```diff
@@ -0,0 +1,919 @@
+    json_object = json_entry_id if json_entry_id else raw_json_string
+    if json_object:
+        return safe_load_json(json_object)
+    return {}
+
+
+def load_content_from_file(entry_id: str) -> str:
+    try:
+        path = demisto.getFilePath(entry_id)
+        with open(path.get('path')) as file:
+            return file.read()
+    except Exception as e:
+        raise ValueError(f'Failed to load the file {entry_id}: {str(e)}')
+
+
+def verify_required_arg(arg_name: str, arg: Optional[str]) -> str:
+    if arg is None:
+        raise ValueError(f'Required {arg_name} argument is missing')
+    return arg
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: CipherTrustClient) -> str:
+    """Tests API connectivity and authentication
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``CipherTrustClient``
+    :param client: CipherTrust client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+    client.get_group_list(params={})
+    return 'ok'
+
+
+def group_list_command(client: CipherTrustClient, args: dict[str, Any]) -> CommandResults:
+    skip, limit = derive_skip_and_limit_for_pagination(args.get(CommandArguments.LIMIT),
+                                                       args.get(CommandArguments.PAGE),
+                                                       args.get(CommandArguments.PAGE_SIZE))
+    params = assign_params(
+        skip=skip,
+        limit=limit,
+        name=args.get(CommandArguments.GROUP_NAME),
+        users=args.get(CommandArguments.USER_ID),
+        connection=args.get(CommandArguments.CONNECTION),
+        clients=args.get(CommandArguments.CLIENT_ID)
+    )
+    raw_response = client.get_group_list(params=params)
+    return CommandResults(
+        outputs_prefix=GROUP_CONTEXT_OUTPUT_PREFIX,
+        outputs=raw_response.get('resources'),
+        raw_response=raw_response,
+        readable_output=tableToMarkdown('group', raw_response.get('resources'))
+    )
+
+
+def group_create_command(client: CipherTrustClient, args: dict[str, Any]) -> CommandResults:
+    request_data = assign_params(name=args.get(CommandArguments.NAME),
+                                 description=args.get(CommandArguments.DESCRIPTION))
+    raw_response = client.create_group(request_data=request_data)
+    return CommandResults(
+        outputs_prefix=GROUP_CONTEXT_OUTPUT_PREFIX,
+        outputs=raw_response,
+        raw_response=raw_response
+    )
+
+
+def group_delete_command(client: CipherTrustClient, args: dict[str, Any]) -> CommandResults:
+    request_data = assign_params(force=args.get(CommandArguments.FORCE))
+    client.delete_group(group_name=verify_required_arg(CommandArguments.GROUP_NAME, args.get(CommandArguments.GROUP_NAME)),
```

Comment:
In all those cases, when I passed the arguments directly the pre-commit tests failed- since the client methods expect a str type and args.get can return Optional. We know that this won't actually happen since those are required arguments, however the tests fail. 
Should I modify the arguments types in the client then to Optional[str]? set a empty string default value for args.get? WDYT?